<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMainWindow</class>
 <widget class="QMainWindow" name="SMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#frame_2{
	background-color: rgb(186, 143, 255);
}
QPushButton{
	background-color: rgb(0,0,0,0);
	color: rgb(255,255,255);
	border: none;

}
QPushButton:pressed{
	padding-top: 5px;
	padding-left:5px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_S_shopping">
            <property name="text">
             <string>订购教材</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_S_order">
            <property name="text">
             <string>我的订购</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_S_update_pw">
            <property name="text">
             <string>修改密码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_login_out">
            <property name="text">
             <string>退出登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>8</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QFrame" name="frame_4">
                <property name="minimumSize">
                 <size>
                  <width>600</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QFrame" name="frame_5">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>1</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit{
	bakground-color: rgb(255,255,255,0);
	border: none;
	border-bottom: 1px solid black;
}
QPushButton{
	background-color: rgb(0,0,0);
	color: rgb(255,255,255);
	border-radius: 7px;
}
QPushButton:pressed{
	padding-top: 5px;
	padding-left:5px;
}</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLineEdit" name="lineEdit_S_M_studentID">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="placeholderText">
                       <string>学号：</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_S_M_bookID">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="placeholderText">
                       <string>书号：下方查看对应书籍的书号</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_S_M_order_sure">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="sizeIncrement">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>确认</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_6">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>8</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QTableView" name="tableView_S_M_bookList"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="minimumSize">
                 <size>
                  <width>600</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QFrame" name="frame_9">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>1</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit{
	bakground-color: rgb(255,255,255,0);
	border: none;
	border-bottom: 1px solid black;
}
QPushButton{
	background-color: rgb(0,0,0);
	color: rgb(255,255,255);
	border-radius: 7px;
}
QPushButton:pressed{
	padding-top: 5px;
	padding-left:5px;
}</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <item>
                     <widget class="QLineEdit" name="lineEdit_S_M_order_update">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="placeholderText">
                       <string>订单号：下方查看对应订单号</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_S_M_order_sure_update">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="sizeIncrement">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>确认删除订单</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_16">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>8</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_13">
                    <item>
                     <widget class="QTableView" name="tableView_2"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <widget class="QFrame" name="frame_8">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>80</y>
                <width>431</width>
                <height>391</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	bakground-color: rgb(255,255,255,0);
	border: none;
	border-bottom: 1px solid black;
}
QPushButton{
	background-color: rgb(0,0,0);
	color: rgb(255,255,255);
	border-radius: 7px;
}
QPushButton:pressed{
	padding-top: 5px;
	padding-left:5px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="lineEdit_S_M_password">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>原密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_S_M_new_password">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>新密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_S_M_new_password_sure">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>确认新密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_S_M_sure">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>确认</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QStackedWidget" name="stackedWidget_2">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page_4"/>
                 <widget class="QWidget" name="page_5">
                  <widget class="QLabel" name="label_4">
                   <property name="geometry">
                    <rect>
                     <x>160</x>
                     <y>0</y>
                     <width>81</width>
                     <height>39</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string>密码不能为空</string>
                   </property>
                  </widget>
                 </widget>
                 <widget class="QWidget" name="page_6">
                  <widget class="QLabel" name="label_5">
                   <property name="geometry">
                    <rect>
                     <x>130</x>
                     <y>10</y>
                     <width>161</width>
                     <height>39</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string>原密码错误或新密码不一致</string>
                   </property>
                  </widget>
                 </widget>
                 <widget class="QWidget" name="page_7">
                  <widget class="QLabel" name="label_6">
                   <property name="geometry">
                    <rect>
                     <x>180</x>
                     <y>0</y>
                     <width>81</width>
                     <height>39</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 182, 38);</string>
                   </property>
                   <property name="text">
                    <string>修改成功</string>
                   </property>
                  </widget>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
